src/main/java/core/gui/theme/GroupTeam.java

```diff
@@ -0,0 +1,31 @@
+package core.gui.theme;
+
+import com.kitfox.svg.app.beans.SVGIcon;
+
+public class GroupTeam {
+
+    private static final String [] FILL_COLOR = {"#FFFFFF", "#f65252", "#d382ff", "#38a0fe","#f4ad2f", "#45e6a4",  "#f6ef6b"};
```

Comment:
Would it make sense to make the colours theme-specific?  For example for Darcula these could have different shades, cf. https://github.com/akasolace/HO/blob/master/src/main/java/core/gui/theme/dark/DarculaDarkTheme.java

Reply:
I try to add colors for light and dark theme, but yes can add different colors if need.